tools/content-generator/build.gradle.kts

```diff
@@ -68,3 +68,5 @@ tasks.named<ShadowJar>("shadowJar") {
     attributes["Main-Class"] = "org.projectnessie.tools.contentgenerator.cli.NessieContentGenerator"
   }
 }
+
+tasks.named<ProcessResources>("processResources") { expand("nessie_version" to project.version) }
```

Comment:
since this was failing missing silently, could it be worth adding an integration test?